corehq/apps/data_interfaces/tasks.py

```diff
@@ -1,13 +1,30 @@
 import uuid
+
 from celery.task import task
+from celery.utils.log import get_task_logger
 from django.core.cache import cache
-from corehq.apps.data_interfaces.utils import add_cases_to_case_group
+from django.template.loader import render_to_string
 from soil import CachedDownload
 
+from corehq.apps.data_interfaces.utils import add_cases_to_case_group, archive_forms
+from dimagi.utils.django.email import send_HTML_email
+
+logger = get_task_logger('data_interfaces')
+ONE_HOUR = 60 * 60
+
 
 @task
 def bulk_upload_cases_to_group(download_id, domain, case_group_id, cases):
     results = add_cases_to_case_group(domain, case_group_id, cases)
     temp_id = uuid.uuid4().hex
-    expiry = 60 * 60
-    cache.set(download_id, results, expiry)
+    cache.set(download_id, results, ONE_HOUR)
+
+
+@task
+def bulk_archive_forms(user, uploaded_data):
+    response = archive_forms(user, uploaded_data)
+    [logger.info("[Data interfaces] {}".format(msg)) for msg in response['errors']]
+    [logger.info("[Data interfaces] {}".format(msg)) for msg in response['success']]
```

Comment:
I know it's more lines, but the standard way to do this is
```python
for msg in response['errors']:
    logger.info("[Data interfaces] %s", msg)
```
Also note the alternate method of string formatting. This allows the logger to delay string formatting to the last possible moment (or avoid it completely if there are no handlers configured at the level, etc.).